pkg/pointer/BUILD.bazel

```diff
@@ -5,4 +5,5 @@ go_library(
     srcs = ["pointer.go"],
     importpath = "kubevirt.io/kubevirt/pkg/pointer",
     visibility = ["//visibility:public"],
+    deps = ["//staging/src/kubevirt.io/api/core/v1:go_default_library"],
```

Comment:
Probably a leftover?

Reply:
Yes, sorry I forgot to commit that change